zip-0207.rst

```diff
@@ -335,14 +254,9 @@ Example implementation
 
```

Comment:
Above the fold the code sets the `startHeight` and `endHeight` for all funding streams to the wrong values, starting at blossom activation and ending at the first halving. It should be made consistent with ZIP 214.